Core/RoverControllerPresets/DriveControllers/DirectDriveController.cs

```diff
@@ -6,6 +6,8 @@ namespace RoverControlApp.Core.RoverControllerPresets.DriveControllers;
 
 public class DirectDriveController : IRoverDriveController
 {
+	private MqttClasses.KinematicMode Mode = MqttClasses.KinematicMode.Ackermann;
```

Comment:
It's private and not a property so its _mode

Reply:
I guess now when it is a property it's fine to leave it like that